Dalamud/Game/Addon/Lifecycle/AddonLifecycleAddressResolver.cs

```diff
@@ -63,6 +65,6 @@ protected override void Setup64Bit(ISigScanner sig)
         this.AddonUpdate = sig.ScanText("FF 90 ?? ?? ?? ?? 40 88 AF");
         this.AddonOnRequestedUpdate = sig.ScanText("FF 90 98 01 00 00 48 8B 5C 24 30 48 83 C4 20");
         this.AddonOnRefresh = sig.ScanText("48 89 5C 24 08 57 48 83 EC 20 41 8B F8 48 8B DA");
-        this.AtkEventListener = sig.GetStaticAddressFromSig("4C 8D 3D ?? ?? ?? ?? 49 8D 8E");
+        this.AtkUnitBase = (AtkUnitBase*)sig.GetStaticAddressFromSig("48 8D 05 ?? ?? ?? ?? 48 89 01 48 8D 05 ?? ?? ?? ?? 4C 89 41 28");
```

Comment:
Can these sigs be moved to CS proper?

Reply:
The callsite sigs can not, as those are for very specific code points.

AddonOnRequestedUpdate is likely already in clientstructs. It's one of the virtual functions for AtkUnitManager.

AtkUnitBase's vtable might already be in clientstructs now that I think about it.

Reply:
It doesn't seem that clientstructs has a address for AtkUnitBase's vtablen from what I can tell

Reply:
I was only able to remove AtkUnitBase, the only other one we can remove is OnRefresh, but currently ClientStructs has the wrong function signature, and that will cause issues, so it's on hold for now.